### PR TITLE
Updating broadcast-cache-update docs

### PR DESCRIPTION
**Pull request by gauntface**
R: @jeffposnick @addyosmani @jpmedley 

I just started to step through the docs to clean up npm-publish-script problems / errors in the jsdoc theme, but in the process fixed up a few quirks / lacking pieces in the docs.

### CODE REVIEW

**Comment by jeffposnick** [comment]
👍  Thanks!

File: packages/sw-broadcast-cache-update/src/index.js [hunk 1]
```diff
@@ -14,7 +14,44 @@
...
+ * The main use of this module will be instantiating a new `Behavior` and
+ * passing it to a
+ * {@link  module:sw-runtime-caching.RequestWrapper|RequestWrapper},
+ * as shown in the first example below.
+ *
+ * You can listen for updates from you web app by adding an event listener on
+ * a browser channel with the same channel name as the Behavior,
+ * which 'cache-updates' in the second example below.
+ *
+ * @example <caption>Using the broadcastCacheUpdate.Behavior class in a
```
Comment (jpmedley) [inline packages/sw-broadcast-cache-update/src/index.js]: I think `broadcastCacheUpdate.Behavior` probably needs backticks or &lt;code&gt; elements.

File: packages/sw-broadcast-cache-update/src/lib/behavior.js [hunk 2]
```diff
@@ -19,30 +19,31 @@ import responsesAreSame from './responses-are-same';
...
+ * and uses the {@link https://developers.google.com/web/updates/2016/09/broadcastchannel|Broadcast Channel API}
+ * to notify interested parties when those Responses differ.
+ *
+ * For efficiency's sake, the underlying response bodies are not compared;
+ * only specific response headers are checked.
  *
- * @example
- * // Used as an automatically invoked as "behavior" by a RequestWrapper:
+ * @example <caption>Added as a "behavior" to a RequestWrapper to
+ * automatically dispatch messages</caption>
```
Comment (jpmedley) [inline packages/sw-broadcast-cache-update/src/lib/behavior.js]: Backticks or code elements.

File: packages/sw-broadcast-cache-update/src/lib/behavior.js [hunk 3]
```diff
@@ -64,27 +65,22 @@ import {defaultHeadersToCheck, defaultSource} from './constants';
...
   /**
-   * Creates a new `Behavior` instance, which is used to compare two
-   * [Responses](https://developer.mozilla.org/en-US/docs/Web/API/Response)
-   * and use the {@link https://developers.google.com/web/updates/2016/09/broadcastchannel|Broadcast Channel API}
-   * to notify interested parties when those Responses differ.
-   *
-   * For efficiency's sake, the underlying response bodies are not compared;
-   * only specific response headers are checked.
+   * To instantiate this Class you must provide a channel name which
+   * will be used to when creating a broadcast channel to dispatch messages on.
```
Comment (jpmedley) [inline packages/sw-broadcast-cache-update/src/lib/behavior.js]: Something went wrong in this sentence.

File: packages/sw-broadcast-cache-update/src/lib/behavior.js [hunk 4]
```diff
@@ -64,27 +65,22 @@ import {defaultHeadersToCheck, defaultSource} from './constants';
...
-   *        the [`BroadcastChannel`](https://developer.mozilla.org/en-US/docs/Web/API/BroadcastChannel/BroadcastChannel).
+   *        the `BroadcastChannel`.
    * @param {Array<string>} input.headersToCheck A list of headers that will be
-   *        used to determine whether the `Response`s differ. If not provided,
-   *        the values `['content-length', 'etag', 'last-modified']` are used.
-   * @param {string} input.source An attribution value that will be used in the
-   *        broadcast message to indicate where the update originated. If not
-   *        provided, a
-   *        {@link constants#defaultSource|default value} will be used.
+   *        used to determine whether the `Response`s differ. Defaults to
```
Comment (jpmedley) [inline packages/sw-broadcast-cache-update/src/lib/behavior.js]: You've been inconsistent about formatting "Response". If it's a specific object use backticks. If you're generically referring to "a response", remove the backticks and lower case the R. Please check throughout.

File: packages/sw-broadcast-cache-update/src/lib/broadcast-update.js [hunk 5]
```diff
@@ -17,18 +17,17 @@ import assert from '../../../../lib/assert';
...
- * to notify interested subscribers about a change to a cached resource.
- *
  * You would not normally call this method directly; it's called automatically
  * by an instance of the {@link Behavior} class. It's exposed here for the
  * benefit of developers who would rather not use the full `Behavior`
  * implementation.
  *
- * The message that's posted takes the following format, inspired by the
+ * Calling this will dispatch a message on the provided {@link https://developers.google.com/web/updates/2016/09/broadcastchannel|Broadcast Channel}
+ * to notifiy interested subscribed about a change to a cached resource.
```
Comment (jpmedley) [inline packages/sw-broadcast-cache-update/src/lib/broadcast-update.js]: Did you mean 'interested subscribers'?

File: packages/sw-broadcast-cache-update/src/lib/behavior.js (hunk 2 above)
Comment (gauntface) [inline packages/sw-broadcast-cache-update/src/lib/behavior.js]: What are you referring to here? If it's caption tag its there as a styled description of the code snippet.

Comment (jpmedley) [inline packages/sw-broadcast-cache-update/src/lib/behavior.js]: I referring to formatting RequestWrapper.

Comment (gauntface) [inline packages/sw-broadcast-cache-update/src/lib/behavior.js]: Done.

File: packages/sw-broadcast-cache-update/src/lib/behavior.js (hunk 3 above)
Comment (gauntface) [inline packages/sw-broadcast-cache-update/src/lib/behavior.js]: I've reworded this sentence - still not great but I'm struggling with what to put.

File: packages/sw-broadcast-cache-update/src/lib/behavior.js (hunk 4 above)
Comment (gauntface) [inline packages/sw-broadcast-cache-update/src/lib/behavior.js]: All lower case unless it links to the Response MDN docs.

File: packages/sw-broadcast-cache-update/src/lib/broadcast-update.js (hunk 5 above)
Comment (gauntface) [inline packages/sw-broadcast-cache-update/src/lib/broadcast-update.js]: indeed I did.

**Comment by gauntface** [comment]
@jpmedley I've had a crack at the comments you've put, feel free to add more changes if you spot anything

**Pull request merged**